File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/logging-json/java/src/test/java/org/acme/logging/json/ExampleResourceTest.java
package org.acme.logging.json;

import io.quarkus.test.junit.QuarkusTest;
import org.hamcrest.Matchers;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.given;
import static org.hamcrest.CoreMatchers.is;

@QuarkusTest
class ExampleResourceTest {

    @Test
    void testHelloEndpoint() {
        given()
          .when().get("/logging-json/")
          .then()
             .statusCode(200)
             .body(is("hello"));
    }

    @Test
    void testGoodbyeEndpoint() {
        given()
                .when().get("/logging-json/goodbye")
                .then()
                .statusCode(500)
                .body(is(Matchers.emptyString()));
    }

}

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/examples/logging-json/kotlin/src/test/kotlin/org/acme/logging/json/ExampleResourceTest.kt
package org.acme.logging.json

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured
import org.hamcrest.CoreMatchers
import org.hamcrest.Matchers
import org.junit.jupiter.api.Test

@QuarkusTest
class ExampleResourceTest {
    @Test
    fun testHelloEndpoint() {
        RestAssured.given()
                .`when`().get("/logging-json/")
                .then()
                .statusCode(200)
                .body(CoreMatchers.`is`("hello"))
    }

    @Test
    fun testGoodbyeEndpoint() {
        RestAssured.given()
                .`when`().get("/logging-json/goodbye")
                .then()
                .statusCode(500)
                .body(CoreMatchers.`is`(Matchers.emptyString()))
    }
}